central/sensor/service/connection/connection_test.go

```diff
@@ -366,7 +457,9 @@ func (s *testSuite) TestDelegatedRegistryConfigOnRun() {
 		clusterMgr:                 mgrMock,
 		delegatedRegistryConfigMgr: deleRegMgr,
 		imageIntegrationMgr:        iiMgr,
+		scanSettingDS:              s.scanConfigDS,
```

Comment:
This shouldn't be needed in this test.

Reply:
The DS is now needed because on connection central sends the scan configuration there